### PR TITLE
Cleanup process wrapping functions

### PR DESCRIPTION
**Pull request by iansf**

### CODE REVIEW

File: lib/src/process.dart [hunk 1]
```diff
@@ -24,3 +26,33 @@ Future<int> runCommandAndStreamOutput(String command, List<String> args) async {
...
+
+/// Run cmd and return stdout.
+String runSync(List<String> cmd) => _runWithLoggingSync(cmd);
+
+String _runWithLoggingSync(List<String> cmd, {bool checked: false}) {
+  _logging.info(cmd.join(' '));
+  ProcessResult results =
+      Process.runSync(cmd[0], cmd.getRange(1, cmd.length).toList());
+  if (results.exitCode != 0) {
+    String errorDescription = 'Error code ' +
```
Comment (devoncarew) [inline lib/src/process.dart]: You may want to try string interpolation here:

```
'Error code ${results.exitCode} returned when attempting to run command: ${cmd.join(' ')}'
```

File: lib/src/process.dart [hunk 2]
```diff
@@ -24,3 +26,33 @@ Future<int> runCommandAndStreamOutput(String command, List<String> args) async {
...
+        results.exitCode.toString() +
+        ' returned when attempting to run command: ' +
+        cmd.join(' ');
+    _logging.fine(errorDescription);
+    if (results.stderr.length > 0) {
+      _logging.info('Errors logged: ' + results.stderr.toString().trim());
```
Comment (devoncarew) [inline lib/src/process.dart]: same - I generally prefer string interpolation, and use concatenation less frequently

**Comment by devoncarew** [comment]
lgtm

File: lib/src/process.dart (hunk 1 above)
Comment (iansf) [inline lib/src/process.dart]: Oh yeah -- when I wrote those logs I didn't know how to make interpolation work with compound statements...

**Pull request merged**